lab-jonathan/package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
This looks like a quick `npm init` when you started the project, but you never came back and updated it, or installed any dependencies. Needs review.